claude/skills/merge-worktree/SKILL.md

```
@@ -0,0 +1,107 @@
+3. After all conflicts resolved:
+   ```bash
+   git -C <MAIN_TREE_PATH> commit --no-edit
+   ```
+
+4. If you cannot confidently resolve a conflict, leave it and tell the user which files need manual attention.
+
+### 5. Mark for Cleanup
+
+After successful merge, tell the user:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)